src/components/OurWork/CaseStudy.js

```diff
@@ -0,0 +1,86 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import StyledLink from '../styledLink'
+import Image from '../Common/Image'
+import { H4, Paragraph } from '../Typography'
+import getIntroSentence from '../../utils/getIntroSentence'
+
+const Services = styled(Paragraph).attrs({ muted: true })`
+  // overwrite default padding on Paragraph component
+  padding-bottom: 0;
+`
+
+const Title = H4.withComponent('h2')
+
+const IntroSentence = styled(Paragraph).attrs({ fullWidth: true })`
+  padding-bottom: 6px;
+  ${breakpoint('tablet')`
+    padding-bottom: 12px;
+  `}
+`
+
+const FlexCol = styled(Col)`
+  display: flex;
+  ${breakpoint('tablet')`
+    align-items: center;
+  `}
+`
+
+const TitleSection = ({ services, title }) => {
+  const commaSeparatedServices = [
+    services.slice(0, -1).join(', '),
+    services.slice(-1)[0]
+  ].join(services.length < 2 ? '' : ' & ')
+  return (
+    <Padding bottom={{ smallPhone: 1, smallTablet: 0 }}>
+      <Services>{commaSeparatedServices}</Services>
+      <Title>{title}</Title>
+    </Padding>
+  )
+}
+
+const InfoSection = ({ introSentence, title, slug }) => (
+  <Fragment>
+    <Padding top={1}>
+      <IntroSentence fullWidth>{introSentence}</IntroSentence>
+    </Padding>
+    <StyledLink
+      aria-label={`Learn more about ${title}`}
+      to={`/case-study/${slug}`}
+    >
+      Learn more
+    </StyledLink>
+  </Fragment>
+)
+
+const CaseStudy = ({ caseStudy }) => {
+  const { title, services, posterImage, slug } = caseStudy
+  const introSentence = getIntroSentence(caseStudy)
+
+  return (
+    <Row>
+      <Col width={[1, 1, 1, 1, 0, 0, 0]}>
+        <TitleSection services={services} title={title} />
+      </Col>
+      <FlexCol width={1}>
+        <Col px={[0]} width={[1, 1, 1, 1, 5 / 12, 4 / 12, 4 / 12]}>
+          <Padding bottom={{ smallPhone: 1, smallTablet: 0 }}>
+            <Image alt={posterImage.title} image={posterImage} />
+          </Padding>
+        </Col>
+        <Col pr={[0]} width={[0, 0, 0, 0, 7 / 12, 6 / 12, 6 / 12]}>
+          <TitleSection services={services} title={title} />
+          <InfoSection introSentence={introSentence} slug={slug} />
+        </Col>
+      </FlexCol>
+      <Col width={[1, 1, 1, 1, 0, 0, 0]}>
```

Comment:
why the width 0 values?

Reply:
On the mobile version (the first 4 values), this section appears - the "InfoSection" that sits below the image.

Whereas [this Col](https://github.com/yldio/yld.io/pull/71/files/c380045678259347ea82310d9571667791bb8612#diff-3a4dcb1f5340f41bba65956ae9faf103R74) has the exact opposite pattern (`width={[0, 0, 0, 0, 7 / 12, 6 / 12, 6 / 12]}`) because it **doesn't** appear on mobile, but **does** appear on small tablet, large tablet & desktop

The downside of this approach is that we're rendering the same content twice & showing/hiding with CSS.

Reply:
yes, this makes it on the browser, in terms of amount of dom elements. Why not putting `display:none` on those breakpoints? that way it's really removed from the DOM, instead of just being hidden by width 0? (I'm assuming it's getting `width: 0` and not `display: none`. If this is wrong, forget what I said :)

Reply:
Yeah, I hear you. When I was pairing with Zooey on the Canon case study, we added [components like `MobileOnlyXYZ`](https://github.com/yldio/yld.io/blob/master/src/pages/case-study/canon-giving-stories-a-new-form.js#L20-L31).

But, if we're going for consistency, it is more common at the moment for us to do it with `width={[0, 0, 0, 0 ....]}` or `width={[...0, 0, 0]} only:
- https://github.com/yldio/yld.io/blob/8d3ff61144e2f1fce0ae2877d833b914623cb015/src/components/Homepage/events.js#L47
- https://github.com/yldio/yld.io/blob/c0c124100fdeb08a4f33967644bec5611ddbe719/src/components/Speciality/projects.js#L52
- https://github.com/yldio/yld.io/blob/c0c124100fdeb08a4f33967644bec5611ddbe719/src/components/Speciality/projects.js#L63
- https://github.com/yldio/yld.io/blob/51714b3fca3b303b0b0d81720f99425dda6617bf/src/components/Speciality/talkToUs.js#L16
etc.

Perhaps this is another case where I should add `display: none` in this situation. But we should raise a ticket to alter the existing codebase to match?

Reply:
[Raised a ticket](https://trello.com/c/4XIDeIsH/263-create-consistent-way-of-showing-hiding-mobile-vs-non-mobile-components) for adding `display: none` elsewhere in the codebase where we're currently relying on width=0 :+1: